arch/riscv/dts/mpfs-polarberry.dts

```diff
@@ -0,0 +1,219 @@
+/dts-v1/;
+
+#include "microchip-mpfs.dtsi"
+
+/* Clock frequency (in Hz) of the rtcclk */
+#define RTCCLK_FREQ		1000000
+
+/ {
+	model = "SundanceDSP Polarberry";
+	compatible = "microchip,mpfs";
```

Comment:
Please use         `compatible = "sundance,polarberry", "microchip,mpfs";` like the linux devicetree